src/lib.rs

```diff
@@ -74,6 +71,9 @@ const ARCH_SYSCALL_PREFIX: &str = "__arm64_";
 #[cfg(target_arch = "x86_64")]
 const ARCH_SYSCALL_PREFIX: &str = "__x64_";
 
+const DEFAULT_BUFFER_CAPACITY: usize = 1024;
+const DEFAULT_ELEMENT_SIZE: usize = 1;
```

Comment:
I feel like this is not a good default size 😆 

We may want to make this required in the builder.

Reply:
It's a very bad default size, yes. But, honestly, every default size here feels wrong. Maybe 32 is a decent compromise, since that's what we use.

Re: making it required, it's only required if the BPF program uses a perfmap. We can't know that until we parse the programs to see which maps they depend on, which happens much later. We also support other kinds of `RWMap`s for getting information in/out of BPF that a user could use instead. If (when 🤞 ) we merge in XDP support we'll have programs without maps at all.

Reply:
Made it so the user has to pass a function that only gets called if a perfmap is found in the program. This avoids us having to have defaults and error checks.